pylua/network.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+import websockets
```

Comment:
@OHerasym, double import. use the second one, just list everything you're using.